resources/src/main/java/org/robolectric/res/android/SortedVector.java

```diff
@@ -20,8 +20,11 @@ public class SortedVector<T extends Comparable<T>> {
   }
 
   public void add(T info) {
-    mStorage.add(info);
-    Collections.sort(mStorage, Comparable::compareTo);
+    int i = Collections.binarySearch(mStorage, info);
+    if (i < 0) {
```

Comment:
I prefer the previous version.

Reply:
I think this new one is fine. Isn't it more efficient (O(log n) search + O(n) insertion), vs (O(n log n) sort ?